.github/workflows/java-editor.yml

```diff
@@ -0,0 +1,55 @@
+name: Java Editor API
+
+on:
+  push:
+    branches:
+      - master
+      - dev
+      - tno-125
+    paths:
+      - api/editor/**
+  pull_request:
+    branches:
+      - master
+      - dev
+    paths:
+      - api/editor/**
+
+jobs:
+  build-api-editor:
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+      working-directory: ./api/editor/api-editor
+
+    strategy:
+      matrix:
+        java-version:
+          - "17.0.1"
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Extract Branch name
+        shell: bash
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+        id: extract_branch
+
+      - name: Use NodeJs ${{ matrix.java-version }}
+        uses: actions/setup-java@v2
+        with:
+          java-version: ${{ matrix.java-version }}
+          distribution: "adopt"
+          cache: maven
+
+      - name: Install Dependencies and Linting
+        run: mvn --batch-mode --update-snapshots verify
+        working-directory: ${{ env.working-directory }}
+
+      # - name: Linting
```

Comment:
Linting can't be turned on until we can fix the IDE to use spaces instead of tabs.  Additionally, linting is applied with Maven build, so it won't need an extra step.